### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by sunshineatnoon** [XinJCheng/CSPN#3]
Hi, could u share the training parameters in the experiment? Such as the learning rate, optimizer, training epochs etc.

**Comment by XinJCheng** [comment]
Thanks for your attention, we have described in the papers, please check the papers in this repo
